migration/migrator/migrations/course/20200313213344_queue_message.py

```diff
@@ -0,0 +1,45 @@
+"""Migration for a given Submitty course database."""
+import json
+from pathlib import Path
+
+def up(config, database, semester, course):
+    """
+    Run up migration.
+
+    :param config: Object holding configuration details about Submitty
+    :type config: migrator.config.Config
+    :param database: Object for interacting with given database for environment
+    :type database: migrator.db.Database
+    :param semester: Semester of the course being migrated
+    :type semester: str
+    :param course: Code of course being migrated
+    :type course: str
+    """
+    course_dir = Path(config.submitty['submitty_data_dir'], 'courses', semester, course)
+    # add boolean to course config
+    config_file = Path(course_dir, 'config', 'config.json')
+    if config_file.is_file():
+        with open(config_file, 'r') as in_file:
+            j = json.load(in_file)
+
+        if 'queue_message' not in j['course_details']:
+            j['course_details']['queue_message'] = 'Welcome to the office hours/lab queue. Choose a queue name and enter the code secret code you have been provided.'
+
+        with open(config_file, 'w') as out_file:
+            json.dump(j, out_file, indent=4)
+
+
+def down(config, database, semester, course):
+    """
```

Comment:
Do the course configuration unit tests still work if you roll back to a previous version without this entry added? If they fail then you should delete the entry `j['course_details']['queue_message']` on a `down` migration.